src/scripts/__tests__/make-webpack-config.spec.ts

```diff
@@ -15,15 +12,17 @@ const styleguideConfig = ({
 	title: 'Style Guide',
 } as unknown) as Rsg.SanitizedStyleguidistConfig;
 
-const getClasses = (plugins: Tapable.Plugin[] = [], name: string): Tapable.Plugin[] =>
+const getClasses = (plugins: any[] = [], name: string): Tapable.Plugin[] =>
 	plugins.filter((x) => x.constructor.name === name);
+
 const getClassNames = (plugins: Tapable.Plugin[] = []): string[] =>
 	plugins.map((x) => x.constructor.name);
 
 const process$env$nodeEnv = process.env.NODE_ENV;
 
 beforeEach(() => {
-	(CopyWebpackPlugin as jest.Mock).mockClear();
+	const mockedPlugin = CopyWebpackPlugin as unknown;
+	(mockedPlugin as jest.Mock).mockClear();
```

Comment:
`as unknown as something` is a common pattern (though the one that should be avoided), and better not to split it into multiple lines.

```suggestion
	((CopyWebpackPlugin as unknown) as jest.Mock).mockClear();
```

Reply:
fixed